src/views/NewDocumentView.js

```diff
@@ -41,31 +40,55 @@ const NewDocumentView = ({match, history, location, createDocument}) => {
   };
 
   return (
-    <GridContainer collapsed={true} className="my-20 px-12">
-      <h3 className="text-blue font-normal m-0 cell-12 row-1">
-        Tell us about your study document
-      </h3>
-      <p className="m-0 cell-12 md:cell-8 row-2">
-        Help ensure the fastest processing and harmonization of your study by
-        telling us about the contents of your uploaded document. This helps our
-        engineers accurately interpret your data.
-      </p>
-      <div className="row-3 cell-3 py-12 Form--Header text-right">
-        Uploaded File:
-      </div>
-      <div className="row-3 cell-9 py-12 Form--Header pl-32">
-        {location.state.file.name}
-      </div>
-      <section className="study-file-list cell-12 row-4">
-        <NewDocumentForm
-          handleSubmit={handleSubmit}
-          handleCancel={() =>
-            history.push(`/study/${match.params.kfId}/documents`)
-          }
-          errors={errors}
+    <Container as={Segment} vertical basic>
+      {errors ? (
+        <Message
+          negative
+          icon="warning circle"
+          header="Error"
+          content={errors}
+        />
+      ) : (
+        <Message
```

Comment:
move this content to a `h3` header and `p` description instead of a `message` as it's essentially the page sub-heading with a description below it